src/Msg.h

```diff
@@ -77,7 +77,7 @@ class Msg {
     return msg;
   }
 
-  static Msg fromKafkaW(std::unique_ptr<KafkaW::Msg> KafkaWMsg) {
+  static Msg fromKafkaW(std::unique_ptr<KafkaW::Msg> &&KafkaWMsg) {
```

Comment:
It looks to me like `fromKafkaW` should take ownership of the the `KafkaWMsg` pointer, and therefore passing it by value makes sense to me, why the change?

Reply:
You're right, that should not have changed